config/test/custom-matchers.ts

```diff
@@ -32,7 +24,7 @@ expect.extend({
                 message: () => `Problem was not fully answered`,
             };
         }
-        if (score.earned !== 1) {
```

Comment:
And incorrect score is one where the `earned` is not equal to `.total`.  Some questions have more than one "point" available.